test/functional/specs/desktop/C2560.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-console */
+import createNetworkLogger from "../../src/networkLogger";
+import gatewayDomain from "../../src/constants/gatewayDomain";
+import { responseStatus } from "../../src/assertions/index";
+
+const networkLogger = createNetworkLogger();
+
+fixture`Sandbox Tests`
+  .page(gatewayDomain)
+  .requestHooks(
+    networkLogger.gatewayEndpointLogs,
+    networkLogger.sandboxEndpointLogs
+  );
+
+test.before(() => console.log("Test Started"));
+test.after(() => console.log("Test Completed"));
+
+const testDescription =
+  "Regression: Given user loads sandbox. The global function named alloy is accessible.";
+
+test.meta({
+  ID: "2560",
+  SEVERITY: "P0",
+  TEST_RUN: "Regression"
+})(testDescription, async () => {
```

Comment:
I don't think the `async` keyword here is useful if you're not `await`ing inside the function.

Reply:
Testcafe fails when I remove the await with the following error: https://github.com/DevExpress/testcafe/issues/2557

Reply:
Oh, I think I understand what's going on now. It seems like you need to add `await` to the line below.